android/app/src/main/java/edu/gatech/cs2340/gtrational/rational/model/User.java

```diff
@@ -18,62 +23,127 @@
     private boolean isLocked;
     private String sessionId;
 
-    public User(String username, String password, String name, String email, PermissionLevel permissionlvl) {
+    private List<String> sessions;
```

Comment:
Also is this user class  supposed to represent the info we have on the user on the client side? If so its good that we have the sessionId above that represents their current sessionId, but I'm not sure if we need a list of sessionIds since that's being stored on the backend